kubernetes/structures_pod.go

```diff
@@ -795,6 +797,14 @@ func expandEmptyDirVolumeSource(l []interface{}) *v1.EmptyDirVolumeSource {
 	obj := &v1.EmptyDirVolumeSource{
 		Medium: v1.StorageMedium(in["medium"].(string)),
 	}
+
+	if v, ok := in["size_limit"].(int); ok {
```

Comment:
`size_limit` is a string type in schema, so I think this will never successfully assert to an int. It might be sufficient to just pass it into `resource.ParseQuantity()` without the `if` statement.
Have you run into any cases where it's passed in as an `int` ?

Reply:
i actually went by another example i found: https://github.com/hashicorp/terraform-provider-kubernetes/blob/master/kubernetes/structures.go#L230-L250

Reply:
nvm, changed an tested. removed int parse logic